Resources/Prototypes/_Midnight/Catalog/Fills/Items/paper.yml

```
@@ -0,0 +1,19 @@
+- type: entity
+  parent: BoxFolderSyndicate
+  id: BoxFolderSyndicateHoldout
+  components:
+  - type: StorageFill
+    contents:
+      - id: PaperHoldoutBriefcase
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)